docs/modules/trino/pages/usage-guide/client-spooling-protocol.adoc

```diff
@@ -0,0 +1,42 @@
+= Client Spooling Protocol
+:description: Enable and configure the Client Spooling Protocol in Trino for efficient handling of large result sets.
+:keywords: client spooling protocol, Trino, large result sets, memory management
+:trino-docs-spooling-url: https://trino.io/docs/476/client/client-protocol.html
+
+The Client Spooling Protocol in Trino is designed to efficiently handle large result sets. When enabled, this protocol allows the Trino server to spool results to external storage systems, reducing memory consumption and improving performance for queries that return large datasets.
+
+For more details, refer to the link:{trino-docs-spooling-url}[Trino documentation on Client Spooling Protocol {external-link-icon}^].
+
+[IMPORTANT]
+====
+The client spooling protocol was introduced in Trino 466 but it only works reliably starting with Trino 476.
```

Comment:
I think a snafu error "not supported" for Trino 451 and 470 would be great. Some customers barely read docs :see_no_evil:

Reply:
470 works but it's flaky. https://github.com/stackabletech/trino-operator/pull/793/commits/ae1dc26c6b82000d4400accbc9d343a4a28004c9

Reply:
we support it as far as Trino supports it and Trino doesn't support it very well on 470.